src/modules/75-cf/hooks/__tests__/useGovernance.test.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+/* eslint-disable jest-no-mock */
+import { renderHook } from '@testing-library/react-hooks'
+import React, { FC } from 'react'
+import { ModalProvider } from '@harness/use-modal'
+import * as useFeatureFlagMock from '@common/hooks/useFeatureFlag'
+import { useGovernance } from '../useGovernance'
+
+jest.mock('services/cf')
+jest.mock('@common/hooks/useFeatureFlag')
+jest.mock('react-router-dom', () => ({
+  useParams: () => jest.fn()
+}))
+
+const renderHookUnderTest = () => {
+  const wrapper: FC = ({ children }) => {
+    return (
+      <div>
+        <ModalProvider>{children}</ModalProvider>
+      </div>
+    )
+  }
```

Comment:
The `div` is superfluous here
```suggestion
  const wrapper: FC = ({ children }) => <ModalProvider>{children}</ModalProvider>
```